packages/neo4j-driver/src/index.js

```diff
@@ -152,8 +152,8 @@ const {
  *       //
  *       // Default value for this option is `false` for backwards compatibility.
  *       //
- *       // **Warning:** `BigInt` doesn't implement the method `toJSON`. In maner of serialize it as `json`, It's needed to add a custom implementation of the `toJSON` on the
- *       // `BigInt.prototype` {@see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#use_within_json}
+ *       // **Warning:** `BigInt` doesn't implement the method `toJSON`. In order to serialize it as `json`, a custom implementation of the method`toJSON` is
```

Comment:
Space missing. Should be _method 'toJSON'_

Reply:
Nice catch